### PR TITLE
add new NSIS logo

### PR DESCRIPTION
**Pull request by idleberg**
***Fixes #[ISSUE]***

**Changes proposed:**
* [x] Add
* [ ] Prepare
* [x] Delete
* [ ] Fix

**Things I've done:**
* [x] I've pulled the latest master branch.
* [x] I've run `npm install` to install all the dependencies.
* [ ] I've run ESLint.
* [ ] My pull request fixes an issue, I referenced the issue.
* [x] I've run `npm run build` to build the extension.

The new NSIS logo will be rolled out in December, but we already started in using it in various places like [GitHub](https://github.com/NSIS-Dev) or [Twitter](https://twitter.com/nsis_tweets). The SVG icon in this pull-request is taken from the [logo repository](https://github.com/idleberg/nsis-logo-v3) I'm maintaining myself.

### CODE REVIEW

**Comment by robertohuertasm** [comment]
It seems that you've missed to change the `supportedExtensions.js` file. You will have to add the `svg: true` attribute to the nsi icon.

**Comment by JimiC** [comment]
@idleberg Thanks for the PR. I suggest we better pick an [official logo](http://nsis.sourceforge.net/New_Logo_Graphics)
It can also give us the ability to have a `dark` and `light` version of the icon.

**Comment by JimiC** [comment]
@robertohuertasm You do realized that the SVG icon size is more than 32x32?

File: icons/file_type_nsi@2x.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...2.732H390.252c-8.42-.282-16.458 2.865-22.455 8.844-5.733 5.72-8.89 13.333-8.89 21.44 0 8.103 3.154 15.71 8.872 21.425l130.907 130.907v1971.933c0 12.28 7.33 23.262 18.76 28.018 3.686 1.516 7.566 2.28 11.537 2.28 7.996 0 15.815-3.243 21.425-8.868l474.672-474.652 474.71 474.65 6.272 6.13h2.655c3.898 1.796 8.142 2.74 12.507 2.74h992.23c12.283 0 23.262-7.33 27.965-18.638 4.75-11.366 2.193-24.346-6.53-33.08zM2343.386 455.912v845.965l-422.96-422.997 422.96-422.968zm-1361.15 562.77L559.28 1441.676V595.703l422.956 422.98zm-23.745-109.5h845.955l-423 422.99L958.49 909.18zm66.598 152.354l422.986 423h-845.96l422.974-423zm496.136 496.137l422.975 423.002h-845.967l422.99-423.002zm-66.646-152.358h845.962l-422.96 422.98-423.002-422.98zm465.85 465.82l422.958-422.972v845.965l-422.96-422.992zm380.096-526.42h-845.946l423.007-422.984 422.94 422.984zm-496.076-496.13h-845.98l422.97-422.972 423.01 422.97zM462.354 413.08h845.933L885.33 836.03 462.354 413.08zm96.922 2020.82v-845.927l299.063 299.104 123.886 123.89-422.95 422.932zm42.837-888.776l845.962.002-422.997 422.994-422.965-422.996zm919.1 919.072L1098.22 2041.27l845.96-.008-422.968 422.934zm73.16 12.55l422.968-422.937 422.978 422.93-845.946.005z"/></svg>
```
Comment (jens1o) [inline icons/file_type_nsi@2x.svg]: ```width="2902.68" height="2902.68"``` 😆

**Comment by idleberg** [comment]
> @idleberg Thanks for the PR. I suggest we better pick an official logo

These are not official logos, but a very early and since rejected draft (by myself) for a new logo. If you prefer, we can postpone this until December, when the new logo will be rolled out in the distrubution and hopefully the new website.

**Comment by JimiC** [comment]
Looks like the logo you provided is already in use, judging by [NSIS tweeter account](https://twitter.com/nsis_tweets). So we are good to go.

**Pull request merged**